src/WebJobs.Script.WebHost/App_Start/WebHostResolver.cs

```diff
@@ -122,19 +124,18 @@ private void EnsureInitialized(WebHostSettings settings)
             {
                 if (_standbyHostManager == null)
                 {
-                    _standbyScriptHostConfig = CreateScriptHostConfiguration(settings);
-
+                    _standbyScriptHostConfig = CreateScriptHostConfiguration(settings, inStandbyMode: true);
+                    StandbyManager.Initialize(_standbyScriptHostConfig);
                     _standbyHostManager = new WebScriptHostManager(_standbyScriptHostConfig, _secretManagerFactory, _eventManager, _settingsManager, settings);
                     _standbyReceiverManager = new WebHookReceiverManager(_standbyHostManager.SecretManager);
+                    InitializeFileSystem();
                 }
             }
         }
 
-        internal static ScriptHostConfiguration CreateScriptHostConfiguration(WebHostSettings settings)
+        internal static ScriptHostConfiguration CreateScriptHostConfiguration(WebHostSettings settings, bool inStandbyMode = false)
```

Comment:
nit: instead of the bool flag, maybe add a CreateStandbyScriptHostConfiguration method that calls this one, then takes and modifies the properties it needs to

Reply:
Done